### PR TITLE
MethodChainingIndentationFixer - nested params bugfix

### PR DESCRIPTION
**Pull request by Slamdunk**
Ref: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3063

Ping @julienfalque

### CODE REVIEW

File: tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php [hunk 1]
```diff
@@ -186,6 +186,24 @@ public function provideFixCases()
 ->bar()
 ;',
             ],
+            [
+                '<?php
+
+    $user->setEmail(\'voff.web@gmail.com\')
```
Comment (keradus) [inline tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php]: use `"` instead of `\'` for readability ;)

Comment (Slamdunk) [reply tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php]: I just copied from other test cases, btw done.

**Comment by keradus** [comment]
could you propose a fix @Slamdunk ?

**Comment by Slamdunk** [comment]
I don't get very well @julienfalque algorithm :\

**Comment by Slamdunk** [comment]
@keradus @julienfalque I've pushed the fix, can you review it please?

**Comment by keradus** [comment]
ping @julienfalque 
could you take a look please? this one is waiting too long

**Comment by Slamdunk** [comment]
Rebased.

Only nightly is failing, I'll ignore it.

Ping @julienfalque :)

**Comment by Slamdunk** [comment]
Ping @julienfalque :\

File: tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php [hunk 2]
```diff
@@ -210,6 +210,32 @@ public function provideFixCases()
         ->method();
 ?>',
             ],
+            [
+                '<?php
+
+    $user->setFoo(1)
+        ->setBar([
+                1 => 1,
```
Comment (julienfalque) [inline tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php]: IMO this line and the one below should be adjusted (removing 4 spaces) as well.

File: tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php [hunk 3]
```diff
@@ -210,6 +210,32 @@ public function provideFixCases()
...
+                ])
+        ->setBaz(true)
+        ->setX(array(
+    2 => 2,
```
Comment (julienfalque) [inline tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php]: Same here: I would add 8 spaces to this line and the one below.

File: tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php (hunk 2 above)
Comment (Slamdunk) [reply tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php]: This PR isn't related to #3558, there is no new feature here.

It's just a bugfix that, if a call contains multiline contents, `MethodChainingIndentationFixer` should fix it as well.

Comment (julienfalque) [reply tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php]: :+1:

**Comment by keradus** [comment]
Thank you @Slamdunk.

**Pull request merged**